assets/src/blocks/amp-story-post-author/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getMetaBlockSettings } from '../../helpers';
+
+export const name = 'amp/amp-story-post-author';
+
+export const settings = {
+	title: __( 'Story Author', 'amp' ),
+	description: __( 'Displays the name of the story\'s author', 'amp' ),
+	category: 'common',
+	icon: 'admin-users',
+	keywords: [
+		__( 'post', 'amp' ),
+		__( 'author', 'amp' ),
```

Comment:
Why would one search for "amp" when wanting to add the byline to an AMP story though?

For the regular post editor a new category can certainly be considered, although this block here is for AMP Stories only. Not sure if that block makes sense for regular posts as well.

Reply:
> Why would one search for "amp" when wanting to add the byline to an AMP story though?

To find all AMP-related blocks? It would be good to have a way how to filter out AMP related blocks only. Thoughts?

Reply:
I see that you opened #1598 for that before. Let's use that for discussion 👍